docs/proposals/mapper-redesign.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Mapper re-design
+status: implementable
+authors:
+    - "@sailorvii"
+    - "@luogangyi"
+approvers:
+  - "@"
+  - "@"
+creation-date: 2020-09-
+last-updated: 
+---
+
+# Mapper Re-design
+## Introduction
+
+The original description could refer to the doc [mapper-design.md](https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/mapper-design.md "mapper-design.md").
+
+Mapper is an interface between kubeedge and devices. It could set/get device data, get and report the device status.
+Kubeedge uses device controller, device twin and mapper to control the devices. 
```

Comment:
```suggestion
KubeEdge uses device controller, device twin and mapper to control the devices. 
```

Reply:
Thanks for your review and time.

Done.